### PR TITLE
task 9

### PR DESCRIPTION
**Pull request by jharold555**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -301,3 +301,30 @@ describe("PATCH /api/articles/:article_id", () => {
...
+describe("DELETE /api/comments/:comment_id", () => {
+  test("returns a 400 error for incorrect id type", async () => {
+    const response = await request(app)
+      .delete("/api/comments/banann")
+      .expect(400);
+    expect(response.body.msg).toBe("400 Bad Request");
+  });
+  test("returns a 404 error for non-existent id", async () => {
+    const response = await request(app).delete("/api/comments/678").expect(404);
+    expect(response.body.msg).toBe("404 comment_id of 678 Not Found");
```
Comment (Hard-Luck) [inline __tests__/app.test.js]: You should be less specific with error messages, for security reasons, people that are would use the api will have the endpoints documentation, so can figure stuff out that way, people that shouldn't be using it should get as little info as possible

**Review by Hard-Luck — CHANGES_REQUESTED** [review]
See slack message

File: db/controllers.js [hunk 2]
```diff
@@ -50,4 +51,10 @@ const patchArticle = (req, res, next) => {
         res.status(200).send({article})
     }).catch(next)
 }
-module.exports = {getTopics, getApis, getArticle, getArticles, getArticleComments, postComment, patchArticle}
\ No newline at end of file
+const deleteComment = (req, res, next) => {
+    const id = req.params.comment_id
+    commentDelete(id).then(() => {
+        res.status(204).send({})
```
Comment (B7Bprog) [inline db/controllers.js:57]: Please remove the empty object argument.

File: __tests__/app.test.js [hunk 3]
```diff
@@ -301,3 +299,27 @@ describe("PATCH /api/articles/:article_id", () => {
...
+      .expect(400);
+    expect(response.body.msg).toBe("400 Bad Request");
+  });
+  test("returns a 404 error for non-existent id", async () => {
+    const response = await request(app).delete("/api/comments/678").expect(404);
+    expect(response.body.msg).toBe("404 Not Found");
+  });
+  test("returns 204 status with no content", async () => {
+    const response = await request(app).delete("/api/comments/8").expect(204);
+    expect(response.body).toEqual({});
```
Comment (B7Bprog) [inline __tests__/app.test.js:315]: This line is not needed. Nothing should be sent back on the 'body'.

**Review by B7Bprog — CHANGES_REQUESTED** [review]
Hey Jasmine, please fix that API response, then it's good to be merged.

**Pull request merged**